### PR TITLE
AO3-4427 Update mysql2 as far as rails will allow

### PR DESCRIPTION
**Pull request by zz9pzza**
I think if this loads its probably ok

https://otwarchive.atlassian.net/browse/AO3-4427

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -172,7 +172,8 @@ GEM
     multi_test (0.1.2)
     multi_xml (0.5.5)
     multipart-post (2.0.0)
-    mysql2 (0.3.10)
+    mysql2 (0.3.20)
+    mysql2 (0.3.20-x86-mingw32)
```
Comment (elzj) [inline Gemfile.lock]: Should this have two entries?

Comment (zz9pzza) [inline Gemfile.lock]: I just run the command to update the gem, It is not the only thing with two entries.

Does it need two, I expect not but I don't want to edit the Gemfile.lock file by hand as that seems wrong.

**Comment by sarken** [comment]
Alas! Merge conflicts.

**Comment by shalott** [comment]
If there were a problem with this I can't imagine our tests wouldn't bomb so I am confident in using it.

**Pull request merged**